src/tests/ftest/pool/RebuildTests.py

```diff
@@ -52,14 +51,40 @@ class RebuildTests(Test):
     def setUp(self):
         self.hostlist = None
         # get paths from the build_vars generated by build
-        with open('../../../.build_vars.json') as f:
-            self.build_paths = json.load(f)
+        with open('../../../.build_vars.json') as file_ds:
+            self.build_paths = json.load(file_ds)
 
         # setup the DAOS python API
-        self.Context = DaosContext(self.build_paths['PREFIX'] + '/lib/')
+        self.context = DaosContext(self.build_paths['PREFIX'] + '/lib/')
+        self.createuid = os.geteuid()
+        self.creategid = os.getegid()
+        # parameters used in pool create that are in yaml
+        self.createmode = self.params.get("mode", '/run/testparams/createmode/')
+        self.createsetid = self.params.get("setname", '/run/testparams/createset/')
+        self.createsize = self.params.get("size", '/run/testparams/createsize/')
+        # how many objects and records are we creating
+        self.objcount = self.params.get("objcount", '/run/testparams/numobjects/*')
+        self.reccount = self.params.get("reccount", '/run/testparams/numrecords/*')
+        if self.objcount == 0:
+            self.reccount = 0
+        # which rank to write to and kill
+        self.rank = self.params.get("rank", '/run/testparams/ranks/*')
+        # how much data to write with each key
+        self.size = self.params.get("size", '/run/testparams/datasize/*')
+        #Start the server
+        self.server_group = self.params.get("server_group", '/server/', 'daos_server')
+        basepath = os.path.normpath(self.build_paths['PREFIX']  + "/../")
+        self.hostlist = self.params.get("test_machines", '/run/hosts/')
+        hostfile = WriteHostFile.WriteHostFile(self.hostlist, self.workdir)
+        ServerUtils.runServer(hostfile, self.server_group, basepath)
 
     def tearDown(self):
-        pass
+        try:
+            # really make sure everything is gone
+	    ServerUtils.killServer(self.hostlist)
+            CheckForPool.CleanupPools(self.hostlist)
```

Comment:
I think we just discussed in the testing channel that this should come after the server shutdown, not before, so swap this line with 87 so that this one is in the finally and the `stopServer()` is in the `try:`

Reply:
It was swapped before but this is rebuild and servers getting killed during tests so after that server stop does not work and even kill does not really kill all the servers after server stop.

Reply:
That sounds like a separate bug then doesn't it?

But I'm not clear why that even means that it's OK to clear out the storage from under the running servers.